src/sql/SqlResult.ts

```diff
@@ -211,7 +212,7 @@ export class SqlResultImpl implements SqlResult {
 
     isRowSet(): Promise<boolean> {
         return this.executeDeferred.promise.then(() => {
-            return this.updateCount === Long.fromInt(-1);
+            return this.rowMetadata !== null;
```

Comment:
just to be sure, `rowMetadata` can't be `undefined`, right?

Reply:
no it cant be